doc/decoders.texi

```diff
@@ -353,6 +353,105 @@ Enabled by default.
 
 @end table
 
+@section aribcaption
+
+Yet another ARIB STD-B24 caption decoder using external @dfn{libaribcaption}
+library.
+
+Implements profiles A and C of the Japanse ARIB STD-B24 standard,
+Brazilian ABNT NBR 15606-1, and Philippines version of ISDB-T.
+
+Requires the presence of the libaribcaption headers and library
+(@url{https://github.com/xqq/libaribcaption}) during configuration.
+You need to explicitly configure the build with @code{--enable-libaribcaption}.
+If both @dfn{libaribb24} and @dfn{aribcaption} are enabled, @dfn{aribcaption}
+decoder precedes.
+
+@subsection aribcaption Decoder Options
+
+@table @option
+
+@item -sub_type @var{subtitle_type}
+Specifies the format of the decoded subtitles.
+
+@table @samp
+@item bitmap
+Graphical image.
+@item ass
+ASS formatted text.
+@item text
+Simple text based output without formatting.
+@end table
+
+The default is @dfn{ass} as same as @dfn{libaribb24} decoder.
+Some present players (e.g., @dfn{mpv}) expect ASS format for ARIB caption.
+
+@item -font @var{font_name[,font_name2,...]}
+Specify comma-separated list of font family names to be used for @dfn{bitmap}
+or @dfn{ass} type subtitle rendering.
+Only first font name is used for @dfn{ass} type subtitle.
+
+If not specified, use internaly defined default font family.
+
+@item -ass_workaround @var{boolean}
+Since some players (e.g., @dfn{mpv}) can't handle multi-rectangle ASS
+subtitle properly, default behavior is all the texts are displayed in
+a single-rectangle at a time.
```

Comment:
What behavior are you seeing in mpv with this set to false? If this isn't working properly, it should be fixed (either in the decoder or in mpv).

Reply:
There is a `mpv` implementation that can handle multi-rectangle ASS subtitle:
https://github.com/0p1pp1/mpv

You can find typical case on https://streams.videolan.org/s
treams/ts/ARIB/japan/osaka_15.ts around 00:03:35.
![osaka_15.ts](https://user-images.githubusercontent.com/19228344/218261058-31b5b752-bae9-4ee3-afeb-49575bddd453.png)
2 captions are displayed on each person at same time.
You can see same scene with `ffplay -sub_type bitmap osaka_15.ts` after this patchset is applied.

Current plain `mpv` cannot handle multi-rectangle ASS subtitle.
If plain `mpv` plays it with `-ass_workaround false`, first caption is immediately clear after second caption is displayed; i.e., only second caption can be seen.

Reply:
![image](https://user-images.githubusercontent.com/819638/218274447-1ef67ca4-be7d-4bf1-a478-f7dc7a218aa3.png)
This works fine in current mpv (7990dd8f3f664e01c984a405dad7c8af320cabf1) for me?

Reply:
Oh, my knowledge is little bit old.
I found mpv release/0.35 can handle multi-rectangle ASS subtitles.

I would like change default value of `ass_workaround` to 0, but stay default definition to be outside of `AVOption`.

Reply:
I don't understand why you want the default to be a macro.

Also, I'd suggest naming the option `single_rect` to clarify its behavior.

Reply:
The reason is explained in latter description (but should amend the explanation to revert default value).

```texi
The default is @var{true}.

If a player can handle multi-rectangle ASS subtitle and you prefer
more suitable positioning, set this option to @var{false} or define
@env{ASS_WORKAROUND=0} to change default behavior at compilation.
```

This can easily change default behavior without change the code.

Reply:
I will change the option name to `ass_single_rect`.